daprdocs/content/en/developing-applications/building-blocks/configuration/howto-manage-configuration.md

```diff
@@ -304,7 +304,7 @@ asyncio.run(executeConfiguration())
 ```
 
 ```bash
-dapr run --app-id orderprocessing --components-path components/ -- python3 OrderProcessingService.py
+dapr run --app-id orderprocessing --resources-path components/ -- python3 OrderProcessingService.py
```

Comment:
```suggestion
dapr run --app-id orderprocessing --resources-path resources/ -- python3 OrderProcessingService.py
```

Reply:
The `components` dir is also correct here but we can change it to `resources` to make it a kind of standard. thoughts ?

Reply:
On thinking again,  it seems correct as of now(until we have the change for `resources dir` merged). Please ignore my previous comment.